packages/ava/src/advisor/utils/index.ts

```diff
@@ -1,5 +1,5 @@
-/**
- * TODO： migrate utils into one unified folder of in ava.
+/*
+ * TODO @GuangMingYouBei: migrate utils into one unified folder of in ava.
```

Comment:
would you finish it before the release of v3? @GuangMingYouBei

Reply:
nope